sign/request.go

```diff
@@ -17,29 +16,14 @@ type Meta interface{}
 
 // Request is a single signing request.
 type Request struct {
```

Comment:
Is it used anywhere?

Reply:
Yes, its used in `sign/notifications.go`. Can't tell if we need that `notifications.go` file at all anymore.

Reply:
We probably don't need it. We can handle it in another PR though. When all PRs are merged to the feature branch, I'd love to play with it so that maybe we can remove even more code.